### DISCUSSION TITLE
Seagate ST-1096N imaged successfully

### DISCUSSION THREAD

**Discussion by XAct-6510** [BlueSCSI/BlueSCSI-v2#214] [General]
This drive came with my "newly" acquired GVP Impact A2000-HC+8 Series II for my Amiga 2000 and will now be replaced by the BlueSCSI. The imaging in Initiator Mode worked flawlessly, see logfile.
[ST1096N-log.txt](https://github.com/user-attachments/files/17906023/ST1096N-log.txt)

**Comment by erichelgeson** [comment]
Thank you for the report! I've added it to the list here - https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Initiator-Mode#working
